recipes/agent_linux.rb

```diff
@@ -1,102 +1,94 @@
-# Installs and configures the New Relic Infrastructure agent on Linux
+#
+# Copyright (c) 2016-2017 New Relic, Inc.
+#
+# All rights reserved.
+#
 
-deb_version_to_codename = {
-  10 => 'buster',
-  9 => 'stretch',
-  8 => 'jessie',
-  7 => 'wheezy',
-  16 => 'xenial',
-  14 => 'trusty',
-  12 => 'precise'
-}
+#
+# Recipe to install and configure the New Relic Infrastructure agent on Linux
+# TODO: Convert to LWRP
+#
+node.default['newrelic_infra']['agent']['flags']['config'] = ::File.join(
+  node['newrelic_infra']['agent']['directory']['path'],
+  node['newrelic_infra']['agent']['config']['file']
+)
 
+# Setup a service account
+poise_service_user node['newrelic_infra']['user']['name'] do
```

Comment:
Is a dependency on poise necessary? I'd prefer to have fewer dependencies, or none at all. If poise buys us something then I might be on board with it, but I'm not sure.

Reply:
Sounds good. I might want to remove them in the future just to avoid the dependencies, but would be fine with merging them as-is. I'm not personally opposed to using Poise, but some consumers of the cookbook might have restrictions on dependencies.